pkg/cmd/server/start/start_node.go

```diff
@@ -280,8 +274,13 @@ func StartNode(nodeConfig configapi.NodeConfig) error {
 
 	config.EnsureVolumeDir()
 	config.EnsureDocker(docker.NewHelper())
+	plugin, endpointFilter := InitSDNPlugin(config, nodeConfig)
```

Comment:
> I don't understand... it's already in the cmd/server/kubernetes package... I just want to make it a field in cmd/server/kubernetes/NodeConfig... why does it need to move?

@liggitt 

> actually, can we move this into BuildKubernetesNodeConfig? I'd prefer config be treated as immutable once BuildKubernetesNodeConfig returns it

Which I took to mean folding InitSDNPlugin() into BuildKubernetesNodeConfig() which actually makes a lot of sense, since there's no real point to having the fields in NodeConfig if they are only set and used in one function (StartNode())?